src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", Helvetica, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+  font-family: inherit;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-weight: 300;
+  height: 70px;
+  padding-left: 62px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input_small {
+  height: 42px;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: url(./images/Search.svg) 26px center / 19px no-repeat, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+  color: #3d4e61;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.input_small:focus {
+  background: url(./images/Search.svg) 13px 15px no-repeat, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
try to add this Search.svg icon as ::before pseudo element or img tag

Reply:
What do you mean 'img tag'? To create <div> with <img> inside?